ppb/events.py

```diff
@@ -1,19 +1,30 @@
 from dataclasses import dataclass
 import logging
 import re
+from typing import Any
 from typing import Collection
+from typing import Dict
 from typing import Set
+from typing import Type
+from typing import Union
 
 from ppb.abc import Scene
 from ppb.buttons import MouseButton
 from ppb.keycodes import KeyCode
 from ppb.vector import Vector
 
 __all__ = (
```

Comment:
FYI, I don't feel a particular need to keep this alphabetical. Thematic sorting is just as reasonable.